src/redux/categories/categories.js

```diff
@@ -0,0 +1,20 @@
+const CHECK_STATUS = './library/categories/CHECK_STATUS';
+
+const stateInit = [];
+
+export const checkStatus = (payload) => ({
+  type: CHECK_STATUS,
+  payload,
+});
+
+const catReducer = (state = stateInit, action) => {
+  switch (action.type) {
+    case CHECK_STATUS:
+      return [...state, action.payload];
+
```

Comment:
- [ ] Good job so far, however, let's simply return a string `"Under construction"` when the action type is `CHECK_STATUS`. Hint; `return "Under construction"`